docs/src/App.jsx

```diff
@@ -3,7 +3,7 @@ var React = require('react');
 var HomePage = require('./HomePage.jsx');
 
 var a11y = require('react-a11y');
-//a11y(React, { throw: true, includeSrcNode: true });
+// a11y(React, {includeSrcNode: true });
```

Comment:
I see a couple of different messages in the console now when I view the docs is this ok?
```
exports You have `role="button"` but did not define an `onKeyDown` handler. Add it, and have the "Space" key do the same thing as an `onClick` handler.
bundle.js:6541 exports You have `role="button"` but did not define an `onKeyDown` handler. Add it, and have the "Enter" key do the same thing as an `onClick` handler.
bundle.js:6541 exports You have an unlabeled element or control. Add `aria-label` or `aria-labelledby` attribute, or put some text in the element.
```